src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -5,13 +5,15 @@ import { Provider } from 'react-redux';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import { Empty } from 'antd';
+import _ from 'lodash';
 import waitForActions from 'redux-mock-store-await-actions';
 import AdvancedFilteringModal from 'components/data-exploration/differential-expression-tool/AdvancedFilteringModal';
 
 import DiffExprResults from 'components/data-exploration/differential-expression-tool/DiffExprResults';
 import { fetchWork } from 'utils/work/fetchWork';
 import { DIFF_EXPR_LOADING, DIFF_EXPR_LOADED, DIFF_EXPR_ORDERING_SET } from 'redux/actionTypes/differentialExpression';
-import '__test__/test-utils/setupTests';
+
+import { mockCellSets1 } from '__test__/test-utils/cellSets.mock';
```

Comment:
don't forget to rename `mockCellSets1` to `mockCellSets`

Reply:
oh right 🤦 changed